protocol/0043-MKTL-market_lifecycle.md

```diff
@@ -301,6 +301,40 @@ Market state is `suspended`.
 The market state is `active`.
 1. When a new governance proposal for "closing" the Spot market, then market state is `trading terminated`.
 
+### Lifecycle happy path in Perpetual market(<a name="0043-MKTL-009" href="#0043-MKTL-009">0043-MKTL-009</a>)
+
+1. Market `m1` is proposed with an internal trading terminated oracle set for some time in the future. Price monitoring is configured (e.g. like `2668-price-monitoring.feature`).
+Market state is `proposed`.
+The LP bond of the party that proposed the market is transferred from general to bond account.
+1. Market `m1` is accepted and enters opening auction.
+Market state is `pending`.
+1. Parties place orders and at least one trade happens in continuous trading mode.
+Market state is `active`.
+1. Parties place orders so that a [price monitoring auction is triggered](0032-PRIM-price_monitoring.md).
+Market state is `suspended`.
+1. Price monitoring auction ends and the market is in continuous trading again.
+The market state is `active`.
+1. Parties cancel orders so that there is no "best static bid" on the order book.
+The market enters [liquidity monitoring auction](0035-LIQM-liquidity_monitoring.md).
+The market state is `suspended`.
+1. A party place bid; this becomes a best static bid.
+After the specified time the liquidity auction ends.
+The market state is `active`.
+1. Make sure that trades happen so that at least two parties have open positions.
+
+1. An oracle event arrives which triggers the perpetual market's interim settlement logic, causing cashflow transfers but the market remains open.
+1. Further trades happen, with parties still having different positions to previously. The mark price is `p`.
+1. The time specified at market proposal by the internal time oracle is reached.
```

Comment:
Again, all of the below will be with a goverance state-change proposal and not oracles.